code/modules/projectiles/ammunition/energy/lmg.dm

```diff
@@ -2,5 +2,4 @@
 	projectile_type = /obj/projectile/bullet/c3d
 	select_name = "spraydown"
 	fire_sound = 'sound/weapons/gunshot_smg.ogg'
-	e_cost = 20
```

Comment:
This is because syndiborgs use this. Change it to `200 WATT` instead of removing it